app/views/vehicles/new.html.erb

```diff
@@ -1,36 +1 @@
-<%= form_with model: [:user, @vehicle], local: true do |form| %>
-
-  <div class="field">
-    <%= form.label :make %>
-    <%= form.text_field :make, required: true %>
-  </div>
-
-  <div class="field">
-    <%= form.label :model %>
-    <%= form.text_field :model, required: true %>
-  </div>
-
-  <div class="field">
-    <%= form.label :year %>
-    <%= form.text_field :year, required: true %>
-  </div>
-
-  <div class="field">
-    <%= form.label :total_seats %>
-    <%= form.number_field :total_seats, placeholder: "   Including Driver", min: 1, required: true %>
-  </div>
-
-  <div class="field">
-    <%= form.label :awd %>
-    <%= form.check_box :awd %>
-  </div>
-
-  <div class="field">
-    <%= form.label :storage_rack %>
-    <%= form.check_box :storage_rack %>
-  </div>
-
-  <div class="actions">
-    <%= form.submit "Add Vehicle" %>
-  </div>
-<% end %>
+<%= render partial: 'form', locals: {path: '/vehicles', method: :post, button_text: 'Add Vehicle'} %>
```

Comment:
I will add an edge case test to make sure that the new vehicle is not associated with another user.